src/api.v2/routes/access.js

```diff
@@ -1,12 +1,26 @@
 const {
-  getExperimentUsers,
+  getUserAccess,
+  inviteUser,
+  revokeAccess,
+  postRegistration,
 } = require('../controllers/accessController');
 
 const { expressAuthorizationMiddleware } = require('../middlewares/authMiddlewares');
 
 module.exports = {
   'access#getExperimentUsers': [
     expressAuthorizationMiddleware,
-    (req, res, next) => getExperimentUsers(req, res).catch(next),
+    (req, res, next) => getUserAccess(req, res).catch(next),
+  ],
+  'access#inviteUser': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => inviteUser(req, res).catch(next),
+  ],
+  'access#revokeAccess': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => revokeAccess(req, res).catch(next),
+  ],
+  'access#postRegistration': [
```

Comment:
So this endpoint is hit by the lambda in SNS?

Reply:
Yes